fmriprep/cli/run.py

```diff
@@ -185,39 +176,20 @@ def get_parser():
         '--reports-only', action='store_true', default=False,
         help='only generate reports, don\'t run workflows. This will only rerun report '
              'aggregation, not reportlet generation for specific nodes.')
+    g_other.add_argument(
+        '--run-uuid', action='store', default=None,
+        help='Specify UUID of previous run, to include error logs in report. '
+             'No effect without --reports-only.')
     g_other.add_argument('--write-graph', action='store_true', default=False,
                          help='Write workflow graph.')
-    
-    # Smoothing options (MCS 10/6/17)
-    g_smoothing = parser.add_argument_group('Specific options for imaging smoothing after preprocessing')
-    g_smoothing.add_argument('--smoothing-kernel',  action='store',
-                        choices = ['4', '6','8','10'], default=6,
-                        help="smoothing width for already preprocessed imaged (default: 6)")
```

Comment:
This looks like you are working in parallel with @jdkent (see poldracklab/fmriprep#706). Again, removed to reduce scope of PR.